auth-rest-api/src/main/java/com/wipro/ats/bdre/md/auth/rest/AuthenticatorAPI.java

```diff
@@ -152,9 +155,9 @@ AuthResult validateToken(@PathVariable("authtoken") String token) {
     }
 
     @RequestMapping(value = "/admin/logout/{authtoken}", method = RequestMethod.DELETE)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "session". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1172)